seacrowd/sea_datasets/wikiann/wikiann.py

```diff
@@ -13,7 +13,7 @@
 _SOURCE_VIEW_NAME = DEFAULT_SOURCE_VIEW_NAME
 _UNIFIED_VIEW_NAME = DEFAULT_SEACROWD_VIEW_NAME
 
-_LANGUAGES = ["ind", "eng", "jav", "min", "sun", "ace", "mly", "map-bms"]
+_LANGUAGES = ["ind", "eng", "jav", "min", "sun", "ace", "mly", "map-bms", "mya", "tgl", "tha", "vie", "khm"]
```

Comment:
Hi, since the code of `mly` is already deprecated based on [this](https://iso639-3.sil.org/code/mly), should we rectify it to either `msa` or `zlm`? It might cause inconsistencies between SEACrowd and NusaCrowd dataloaders, tho (if possible we shd rectify it in NusaCrowd asw). 

I'm more keen to change into `msa` since it's a Macrolang and we don't really know the region of collections (hence `msa` is safer than `zlm`)

wdyt, @SamuelCahyawijaya?

```suggestion
_LANGUAGES = ["ind", "eng", "jav", "min", "sun", "ace", "msa", "map-bms", "mya", "tgl", "tha", "vie", "khm"]
```